cli/src/config.rs

```diff
@@ -77,6 +81,16 @@ impl Config {
     pub fn set_ignore_certs_cli(&mut self, ignore_certs_cli: bool) {
         self.ignore_certs_cli = ignore_certs_cli;
     }
+
+    /// Write updates to the configuration file.
+    pub fn save(&self) -> Result<()> {
+        let path = match &self.path {
+            Some(path) => path,
+            None => return Ok(()),
+        };
+
+        save_config(path, self)
```

Comment:
Yeah that's quite interesting actually: We use this to write `.phylum_project`. It makes sense since it basically is just an atomic serialization utility, but maybe the name isn't ideal.